### ISSUE TITLE
Cards should not be dealt right after the user joins the table

### ISSUE THREAD

**Issue by piggydoughnut** [chips-blockchain/bet#137]
We should get rid of dealing cards after joining the table and let the dealer pres start to deal cards for the game. Thoughts?

**Comment by norbertdragan** [comment]
Yes, that should not happen. Cards should be dealt based on when the players are receiving their card info from the backend.
